location/templates/apartment_update.html

```diff
@@ -7,7 +8,12 @@
 <span>Edit</span>
 {% endblock %}
 
-{% load crispy_forms_tags %}
+{% block script %}
+<!-- Allows us to show the Modal -->
```

Comment:
Nice!
I was wondering why we need this bootstrap import because I thought we already had similar one in the `nav.html`. But I found that we only have css import and not js

Reply:
Yes, I made the same mistake when working with the modal window since I had not imported js. Good idea putting the comment in there!